util.go

```diff
@@ -9,3 +15,21 @@ func sliceContains(group []string, name string) bool {
 
 	return false
 }
+
+// WriteFile writes the buffer to the file.
+func WriteFile(reader io.Reader, path string) error {
+	dirPath := filepath.Dir(path)
+	if err := os.MkdirAll(dirPath, os.ModeDir|os.ModePerm); err != nil {
+		return err
+	}
+
+	file, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
I think there's an os.Create that does roughly this.

Reply:
Changed to os.Create